CONTRIBUTING.md

```diff
@@ -45,7 +45,7 @@ $ go run -tags ui ./cmd/all-in-one/main.go
 Alternatively, the path to the built UI assets can be provided via `--query.static-files` flag:
 
 ```
-$ go run ./cmd/all-in-one/main.go --query.static-files jaeger-ui/build
+$ go run ./cmd/all-in-one/main.go --query.static-files jaeger-ui/packages/jaeger-ui/build
```

Comment:
We actually have a make target now that can be used instead: `make run-all-in-one`

Reply:
Ah okay, glad you pointed this out. Do you think we should delete this section of the contrib file then (i.e. "Alternatively, ..." + code block)?

Moreover, given we already have the `make build-ui` instruction in the previous section, could we just have a one-liner for this "Running local build with the UI" section of:
```
make run-all-in-one
```

Unless if we feel the first two sentences (i.e. "The jaeger-ui... .gitignored-ed.") are useful information.

Reply:
I think so, the whole section here could be replaced with this one make target.